tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -931,10 +931,9 @@ ColumnNames_t GetValidatedColumnNames(RLoopManager &lm, const unsigned int nColu
       // Look for a possible overlap between the unknown columns and the
       // columns we should ignore for the purpose of the following exception
       std::set<std::string> intersection;
-      auto colsToIgnore = lm.GetSuppressErrorsForMissingBranches();
+      auto &&colsToIgnore = lm.GetSuppressErrorsForMissingBranches();
```

Comment:
Why 
```suggestion
      auto &&colsToIgnore = lm.GetSuppressErrorsForMissingBranches();
```
rather than:
```suggestion
      const auto &colsToIgnore = lm.GetSuppressErrorsForMissingBranches();
```

Reply:
No particular reason, it won't matter much for the purpose of passing it to `std::set_intersection`, I will change it in the next commit

Reply:
Related post: https://stackoverflow.com/questions/13130708/what-is-the-advantage-of-using-forwarding-references-in-range-based-for-loops/13130795#13130795